packages/@aws-cdk/service-spec-build/src/import-cloudformation-registry.ts

```diff
@@ -26,6 +26,7 @@ export function importCloudFormationRegistryResource(options: LoadCloudFormation
   const specBuilder = new SpecBuilder(db);
   const resourceBuilder = specBuilder.resourceBuilder(resource.typeName, {
     description: resource.description,
+    primaryIdentifier: resource.primaryIdentifier,
```

Comment:
Please translate the primaryIdentifier from `/properties/WorkspaceId` to `WorkspaceId`.